Headers/DebugServer2/Utils/CompilerSupport.h

```diff
@@ -59,6 +59,18 @@
 #define DS2_ATTRIBUTE_NORETURN
 #endif
 
+#if __has_attribute(packed)
+#define DS2_ATTRIBUTE_PACKED __attribute__((packed))
+#else
+#define DS2_ATTRIBUTE_PACKED
```

Comment:
I think you should `#error` if these attributes are not available with the compiler being used. If we just build without packing these structures, we're pretty much guaranteed to have a buggy debug server and that's gonna lead to very hard to track down bugs.